### PR TITLE
Add devcontainer.json configuration

### PR DESCRIPTION
**Pull request by ark120202**

### CODE REVIEW

File: .devcontainer/devcontainer.json [hunk 1]
```diff
@@ -0,0 +1,14 @@
+{
+    "name": "TypeScriptToLua",
+    "image": "mcr.microsoft.com/vscode/devcontainers/javascript-node:14",
+    "settings": {
+        "terminal.integrated.shell.linux": "/bin/bash"
+    },
+    "extensions": [
+        "ark120202.vscode-typescript-to-lua",
```
Comment (Perryvw) [inline .devcontainer/devcontainer.json:8]: Why is this needed to develop tstl? Isn't this only for when you want to use it?

Comment (ark120202) [reply .devcontainer/devcontainer.json:8]: We have 2 lua projects in this repo - lualib and benchmark
